plugins/package-managers/python/src/main/kotlin/Pipenv.kt

```diff
@@ -60,6 +60,9 @@ internal object PipenvCommand : CommandLineTool {
 class Pipenv(
     override val descriptor: PluginDescriptor = PipenvFactory.descriptor, private val config: PipConfig
 ) : PackageManager("Pipenv") {
+    // Usually, definition files should not be limited to lockfiles to also support the case when no lockfile is
```

Comment:
if we define a lockfile to be something different than a definition file, then saying "definition files should not be limited to lockfiles" is a bit odd. 

Could it say "Usually, the globs should not be limited to lockfiles..."

Reply:
I've changed the wording to "definition files should not contain (only) lockfiles", which resembles the wording used in the message of the first commit in this PR.